packages/extension-ui/src/partials/DerivationPath.tsx

```diff
@@ -21,7 +21,7 @@ interface Props {
 
 function DerivationPath ({ className, defaultPath, onChange, parentAddress, parentPassword }: Props): React.ReactElement<Props> {
   const [path, setPath] = useState<string>(defaultPath);
-  const [showAdvancedOptions, setShowAdvancedOptions] = useState(false);
+  const [showPathInput, setShowPathInput] = useState(false);
```

Comment:
```suggestion
  const [enablePathInput, setEnablePathInput] = useState(false);
```

Reply:
Or maybe even more straightforward: `isDisabled` / `isLocked`. And then change usages accordingly (eg. instead of `isExpanded` we'd have `isUnlocked` or `isEditabled`).